File: java_law/it/unimi/di/big/mg4j/index/snowball/Among.java
package it.unimi.di.big.mg4j.index.snowball;

//import java.lang.reflect.Method;

public class Among {
	public Among( String s, int substring_i, int result ) {
		this.s_size = s.length();
		this.s = s.toCharArray();
		this.substring_i = substring_i;
		this.result = result;
	}

	public final int s_size; /* search string */

	public final char[] s; /* search string */

	public final int substring_i; /* index to longest matching substring */

	public final int result; /* result of the lookup */
}

File: python/snowballstemmer/among.py
from typing import Callable, Optional

class Among:
    def __init__(self, s: str, substring_i: int, result: int, method: Optional[Callable[..., bool]] = None) -> None:
        """
        @ivar s search string
        @ivar substring index to longest matching substring
        @ivar result of the lookup
        @ivar method method to use if substring matches
        """
        self.s = s
        self.substring_i = substring_i
        self.result = result
        self.method = method
